frontend/src/components/MeetingRoom/Toolbar/Toolbar.tsx

```diff
@@ -86,7 +87,10 @@ const Toolbar = ({
               isSharingScreen={isSharingScreen}
               isViewingScreenShare={isViewingScreenShare}
             />
-            <LayoutButton isScreenSharePresent={isScreenSharePresent} />
+            <LayoutButton
+              isScreenSharePresent={isScreenSharePresent}
+              isPinnedPresent={isPinnedPresent}
+            />
```

Comment:
Note: I consider combining these two into something like this:

```
<LayoutButton
              isDisabled={isScreenSharePresent || isPinnedPresent}
/>
however, keeping the props separately helps with handling with cases when there are both screen share and pinning present and it helps with the tooltip title in `LayoutButton` since if I just pass in `isDisabled` I won't be able to differentiate what title to display 🙏

Reply:
I think the combined approach better, since the component should only care about whether it is active or not.  That said, I see that the component uses the the split to handle more accurate error messages.  Even so, we can refactor to decouple the error message from the component too 🤷‍♂️ 

In any case, not asking for any changes.  Let's see how others feel, please.